.clang-format

```diff
@@ -24,5 +24,8 @@ Standard:  Cpp11
 AllowAllParametersOfDeclarationOnNextLine: true
 BinPackParameters: false
 BinPackArguments: false
+---
+Language: Proto
+# Don't format .proto files.
```

Comment:
Why not format .proto file?

Reply:
Yep. Will add benchmark in next PR to decide which to use. That won't affect current code structure.